README.md

```diff
@@ -195,9 +195,8 @@ a single `Object`, like `a.b.c.d`. Its corresponding s-expression would be:
 You can also search using nested arrays with **pure values**, or **shortcuts** or
 **procs**:
 
-    Fast.match?(ast, [:send, [:send, '...'], :d]) # => true
-    Fast.match?(ast, [:send, [:send, '...'], :c]) # => false
-    Fast.match?(ast, [:send, [:send, [:send, '...'], :c], :d]) # => true
+    Fast.match? ast, [:send, [:send, '...'], :d]  # => true
+    Fast.match? ast, [:send, [:send, '...'], :c]  # => false
```

Comment:
In the previous examples you have `pattern, ast`, but in this one it's kept reversed?

Reply:
good catch 😎 Let me fix it.